pybamm/util.py

```diff
@@ -271,7 +271,14 @@ def get_parameters_filepath(path):
 
 
 def have_jax():
-    """Check if jax and jaxlib are installed with the correct versions"""
+    """
+    Check if jax and jaxlib are installed with the correct versions
+
+    Returns
+    -------
+    bool
+
```

Comment:
Please add description of what the return is (e.g. Returns True if jax and jaxlib are instaleld with the correct versions and False otherwise)